tests/test_osslsigncode.py

```diff
@@ -64,11 +64,12 @@ async def test_sign_file(test_file, digest_algo, tmp_path, signing_keys):
     priv_key = load_private_key(open(signing_keys[0], "rb").read())
     certs = load_pem_certs(signing_keys[1].read_bytes())
     cafile = signing_keys[1]
+    tsfile = '/etc/ssl/certs/ca-certifcates.crt'
```

Comment:
Are you misspelling this everywhere (certificates) or is it misspelled on disk?
(Same spelling below multiple times; I'm guessing it's a misspelling that we've mocked out in the test run)